apps/formbricks-com/app/docs/contributing/setup/page.mdx

```diff
@@ -90,6 +90,8 @@ To get the project running locally on your machine you need to have the followin
 
    **You can now access the Formbricks app on [http://localhost:3000](http://localhost:3000)**. You will be automatically redirected to the login. To use your local installation of formbricks, create a new account.
 
+   <Note>There are no default values or default accounts in general. You can just create a new account and proceed.</Note>
```

Comment:
Can you change this to
`A fresh setup does not have a default account. Please create a new account and proceed accordingly.`